layouts/shortcodes/featured/scroll-video.html

```diff
@@ -0,0 +1,24 @@
+{{ $hed := .Get "hed" }}
+{{ $dek := .Get "dek" }}
+{{ $outro := .Get "outro" }}
+
+
+<div class="mx-[calc(-50vw_+_50%)] -mt-12 bg-black px-8 py-12 text-white">
```

Comment:
I thought at some point I had a special class for -50vw + 50%. It might have been back in the old Bulma CSS. It's useful enough that maybe it should be added to the Tailwind config or tw.css the next time we need it.